native/cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -60,15 +67,23 @@ using namespace cc; //NOLINT
 namespace {
 AudioEngineImpl *gAudioImpl         = nullptr;
 int              outputSampleRate   = 44100;
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 int              bufferSizeInFrames = 192;
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+// TODO(hack) : There is currently a bug in the opensles module,
+// so openharmony must configure a fixed size, otherwise the callback will be suspended
+int              bufferSizeInFrames = 2048;
+#endif
 
 void getAudioInfo() {
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
```

Comment:
On openharmony it will set nothing?

Reply:
yes，openharmony does not have these interfaces